src/NumericalAlgorithms/RootFinding/QuadraticEquation.hpp

```diff
@@ -17,6 +17,29 @@
  */
 double positive_root(double a, double b, double c) noexcept;
 
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Returns the smallest root of a quadratic equation \f$ax^2 +
+ * bx + c = 0\f$ that is greater than the given value, within roundoff.
+ * \returns A root of a quadratic equation.
+ * \requires That there are two real roots.
+ */
+template <typename T>
+T smallest_root_greater_than_value_within_roundoff(const T& a, const T& b,
+                                                   const T& c,
+                                                   double value) noexcept;
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Returns the largest root of a quadratic equation
+ * \f$ax^2 + bx + c = 0\f$ that is between min_value and max_value,
+ * within roundoff.
+ * \returns A root of a quadratic equation.
+ * \requires That there are two real roots.
```

Comment:
Doesn't this also require that at least one of the roots is between min_value and max_value, because if not, it gives a runtime error? I wonder if you should warn users that this will give a runtime error on failure